backends/bmv2/parser.cpp

```diff
@@ -232,15 +232,18 @@ unsigned ParserConverter::combine(const IR::Expression* keySet,
             int width = type->width_bits();
             BUG_CHECK(width > 0, "%1%: unknown width", e);
 
-            mpz_class key_value, mask_value;
+            mpz_class key_value, mask_value, mask_value2;
```

Comment:
Please comment this code. It is not immediately obvious what mask_value2 means, unless one refers back to the issue.

Reply:
The extra local variable was not strictly necessary, so I removed it, as well as adding a comment explaining the new case.